pkgdiff

```diff
@@ -22,4 +28,18 @@ ebuild "${1}" unpack
 ebuild "${2}" unpack
 s1=$(sed -nr 's/^declare -x S="(.*)"/\1/p' "${PORTAGE_TMPDIR}"/portage/${cat1}/${fn1%.ebuild}/temp/environment)
 s2=$(sed -nr 's/^declare -x S="(.*)"/\1/p' "${PORTAGE_TMPDIR}"/portage/${cat2}/${fn2%.ebuild}/temp/environment)
-diff --color=always --strip-trailing-cr -dupNr "${s1}" "${s2}" | ${PAGER:-less}
+
+{
+	if [[ $build_system -eq 1 ]]; then
+		find "${s1}" "${s2}" -type f \
+			! \( \
+					-name 'configure.ac' -o \
+					-name 'meson.build' -o \
+					-name 'meson_options.txt' -o \
+					-name '*.cmake' -o \
+					-name 'CMakeLists.txt' \
+			\) -printf "%f\n" | diff --color=always --strip-trailing-cr -X - -dupNr "${s1}" "${s2}"
+	else
+		diff --color=always --strip-trailing-cr -dupNr "${s1}" "${s2}"
```

Comment:
Perhaps you could move `diff` out of the scope, and just pass an empty list in standard mode?

Reply:
Yep, looks like I don't really even need to do anything special.